Accepted-RFCs/RFC-012-internal-knowledge-base.md

```diff
@@ -0,0 +1,121 @@
+# RFC-012: Internal Knowledge Base
+## Proposed by
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X))
+
+Initially submitted on 24 Aug 2022
+
+## Current status
+
+- [x] Draft
+- [x] Under discussion
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+## Proposal overview
+
+We need a central knowledge repository for contributors that:
+* Isn't fragmented across different platforms or accounts
+* Is easy for new contributors to access
+* Has a sense of hierarchy
+* Allows contributors to dump critical or needed info in a place that keeps them from being a bus number
+* Has some sense of discipline/oversight to keep it from becoming disorganized or discordant
+
+This is internal-facing material, which is about the audience, and not about privacy. We should assume all information is publicly viewable, just intended to be viewed by those with working group context. Our template/product users wouldn't need to know this stuff, but there's no reason to hide it.
+
+## Motivation
+
+We don't have a good way to share information to use contributors, or a common places where information can live and grow. This was mentioned by multiple people during our recent retrospectives.
+
+## Proposal
+
+We could use a GitLab wiki or repo as a place where contributors can post material or comment on said material with questions, corrections, etc.
+
+We'd have some guideline on structure, intent, etc. Each core initiative would have a space for their information, along with general information, community resources, etc.
+
+Ideally, our docs would make use of our templates—in fact, they would make for great use/test cases. However, we would allow for a "quickly written" status, in cases where information needs to be dumped from someone's mind, but the time/effort cost in polishing isn't currently possible.
+
+We'd have a mechanism for checking the freshness of content, though we can defer the details of that until later.
+
+### GitLab wiki: a.k.a. Do we already have this?
+There's a wiki in GitLab, and [Aaron has release weight information there](https://gitlab.com/tgdp/governance/-/wikis/Guide-to-assigning-weight-scores-to-issues-and-epics-%28release-planning%29). Should we just go with that? Does that make sense for us as a group of groups?
+
+Is the wiki going to be oversaturated with other purposes? Would it be difficult to use search functionality when looking for KB articles, due to non-KB stuff in the wiki also populating the search results?
```

Comment:
Can you elaborate on what you think could be saved in the wiki that would qualify as non-KB stuff? If we can pinpoint certain things, we can brainstorm with the CS group where those things should live.

Reply:
This is a general question regarding technological limitations. Ideally there should be only KB material in whatever CMS or repo we use, but if we're not able to accomplish that, how to do we handle such a situation?